kornia/utils/image.py

```diff
@@ -47,16 +47,15 @@ def _to_bchw(tensor: torch.Tensor) -> torch.Tensor:
     """Converts a PyTorch tensor image to BCHW format.
 
     Args:
-        tensor (torch.Tensor): image of the form :math:`(H, W)`, :math:`(C, H, W)`, :math:`(H, W, C)` or
-            :math:`(B, C, H, W)`.
+        tensor (torch.Tensor): image of the form :math:`(*, H, W)`.
```

Comment:
```suggestion
        tensor: image of the form :math:`(*, H, W)`.
```
typing is automatically generated now (don't fix all) I'll do the missing in separate PR